client/src/components/ReportCollection.js

```diff
@@ -22,9 +22,9 @@ export const FORMAT_MAP = "map"
 export const GQL_REPORT_FIELDS = /* GraphQL */ `
   uuid, intent, engagementDate, duration, keyOutcomes, nextSteps, cancelledReason
   atmosphere, atmosphereDetails, state
-  author { uuid, name, rank, role }
-  primaryAdvisor { uuid, name, rank, role },
-  primaryPrincipal { uuid, name, rank, role },
+  author { uuid, name, rank, role, avatar(size: 32) }
```

Comment:
Is it wise to get the avatar for these three in a report collection? Potentially this could be *lot* of data.

Reply:
It is a fair point, however not showing the avatars would break the style consistency with other parts. A way to go around this would be to cache the avatar data, but this seems a lot of hassle for little gain. Another option may be to retrieve the reports (and avatars) in batches (for each page). My suggestion is to just show the avatar (and maybe more user info) on mouse-over in data heavy places like this.

Showing a smaller avatar or a stretched smaller avatar to save data is not really an option.

Maybe @VassilIordanov could give some input on this issue?

Reply:
I would say that ultimately  - yes, we want to have the avatars of these 3 when we render paginated results. However, with the way ReportCollection is implemented now, it may be prudent to disable this until we replace that class.

Reply:
Ok, I will disable them for now (until the class is replaced)